src/store/compressors.rs

```diff
@@ -13,6 +13,9 @@ pub trait StoreCompressor {
 /// The default is Lz4Block, but also depends on the enabled feature flags.
 #[derive(Clone, Debug, Copy, PartialEq, Eq, Serialize, Deserialize)]
 pub enum Compressor {
+    #[serde(rename = "noop")]
+    /// No-op compressor
+    NoOp,
```

Comment:
Can we rename this `NoCompression`

Reply:
How about just `None`, serialized as `"none"`? The other variants don't have a `Compression` suffix and are serialized as `"lz4"`, `"snappy"`, `"brotli"`.

Reply:
ok